### DISCUSSION TITLE
GPU usage Mac M1

### DISCUSSION THREAD

**Discussion by artincube** [photoprism/photoprism#3530] [Q&A]
using Activity Monitor I can see no GPU usage by Virtual Machine Service, while CPU is nearing 800% (of 8 cores) during  a thumbs regeneration cli
docker compose exec photoprism photoprism thumbs -f

I have
PHOTOPRISM_FFMPEG_ENCODER: "apple" 
enabled in the compose yml file.

Anything else I should do?

**Comment by lastzero** [comment]
You may need to share the corresponding device. Unfortunately, we are on vacation and can't look it up for you.

**Reply by artincube** [reply]
I'll search a bit more while I wait.

**Reply by lastzero** [reply]
Also possible that it only works if you run it locally, so without Docker.

**Reply by artincube** [reply]
Oh ok. I haven't checked local installation, went straight for docker. I'll check that too.

**Comment by artincube** [comment]
> Also possible that it only works if you run it locally, so without Docker.
I looked for this. Is there any way to install without Docker on a Mac? Can't find.

Is there a link for this?

**Reply by lastzero** [reply]
You'd need to compile it. However, there's no native TensorFlow library available for the M1 yet and we can't provide it as we don't don't have the hardware required. We accept hardware donations though.

**Reply by artincube** [reply]
I hear you.
